packages/typedoc-plugins/__tests__/tag-event/fixtures/eventsvalid.ts

```
@@ -0,0 +1,200 @@
+ * @deprecated
+ */
+export type EventFooWithParams = {
+    name: string;
+    args: [p1: string, p2: number, p3: ExampleType];
+};
+
+/**
+ * @eventName ~EventsValidClass#event-foo-no-content
+ */
+// eslint-disable-next-line @typescript-eslint/ban-types
+export type EventFooNoContent = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)